### ISSUE TITLE
Wrong denominators when normalize xx_channel and yy_channel?

### ISSUE THREAD

**Issue by jiayily** [uber-research/CoordConv#5]
Hi, in your code you normalize xx_channel and yy_channel as following:
https://github.com/uber-research/CoordConv/blob/27fab8b86efac87c262c7c596a0c384b83c9d806/CoordConv.py#L70
https://github.com/uber-research/CoordConv/blob/27fab8b86efac87c262c7c596a0c384b83c9d806/CoordConv.py#L71
I think it should be:
`xx_channel = tf.cast(xx_channel, 'float32') / (self.y_dim - 1)`
`yy_channel = tf.cast(yy_channel, 'float32') / (self.x_dim - 1)`
That is the denominators are wrong?

**Comment by jeanibarz** [comment]
Yes it is, there are already 2 pull requests for that issue (one created since 2020). I think that the repository is not maintained anymore.
